src/fontra/views/editor/visualization-layer-definitions.js

```diff
@@ -851,6 +981,117 @@ registerVisualizationLayerDefinition({
   },
 });
 
+registerVisualizationLayerDefinition({
```

Comment:
The two guideline vis layers should be next to each other. Why are they so far apart?

Reply:
Thank you very much for the hint. Obviously I made it already wrong with anchors. I moved it (.selected.guidelines + .selected.anchors)